scripts/common/get-node.sh

```diff
@@ -10,6 +10,17 @@ rm -rf $ANSIBLE_HOSTS
 
 # Get nodes
 nodes=$(cico -q node get --count ${NODE_COUNT} --column hostname --column ip_address --column comment -f value)
+cico_ret=$?
+
+# Fail in case cico returned an error, or no nodes at all
+if [ ${cico_ret} -ne 0 ]
+then
+    echo "cico returned an error (${cico_ret})" >/dev/stderr
```

Comment:
Isn't it more idomatic to redirct for error via: `>&2`
Like: `echo "foo" >&2`

Reply:
Either works. Not sure if this repository has a preferred way of doing it.

Reply:
Is there really a reason to use stderr?

Reply:
Isnt that what always should be done for error messages?

Reply:
@nigelbabu do you want me to drop the redirect to stderr, or do you prefer the `>&2` notation?

Reply:
I would prefer you dropped the redirect to stderr entirely.

Reply:
Ok, dropped the redirection and added the missing `then`. Please have a look again and let me know if there is something else that needs improving.

Thanks!